python/cudf/cudf/core/column/lists.py

```diff
@@ -20,6 +21,43 @@ def __init__(
             null_count=null_count,
             children=children,
         )
+        self._cached_size = None
```

Comment:
Should we move this `_cached_size` attribute into the `ColumnBase` object instead? No reason we can't cache it for Fixed-Width columns as well.

Reply:
Pushed it to parent `Column` implementation and using it in `StringColumn` as well.